framework/src/splits/Split.C

```diff
@@ -84,8 +84,8 @@ Split::Split(const InputParameters & parameters)
     _schur_ainv(getParam<MooseEnum>("schur_ainv"))
 {
   _petsc_options.flags = getParam<MultiMooseEnum>("petsc_options");
-  _petsc_options.inames = getParam<std::vector<std::string>>("petsc_options_iname");
-  _petsc_options.values = getParam<std::vector<std::string>>("petsc_options_value");
+  _petsc_options.pairs =
+      getParamPairs<std::string, std::string>("petsc_options_iname", "petsc_options_value");
```

Comment:
Could we change `getParamPairs`  to `getParam`? We should know it is handling pairs from template parameters